### PR TITLE
Feature/sync gateway sanity nightly tagging

### PR DESCRIPTION
**Pull request by sethrosetter**
- Add nightly tag to tests taking longer than 3 min
- Add template for robot framework tests

### CODE REVIEW

File: _queue/template.robot [hunk 1]
```diff
@@ -0,0 +1,48 @@
+*** Settings ***
```
Comment (tleyden) [inline _queue/template.robot]: what is the meaning of the directory name `_queue`?

Comment (sethrosetter) [inline _queue/template.robot]: I would like to provide a place for dev to put robot testcase definition to allow me to triage them and implement them. Maybe it needs a better name?

Comment (tleyden) [inline _queue/template.robot]: Ok, makes sense.  I think queue is a bit confusing, what about calling it:

- `templates`
- `unimplemented`
- `in_progress`
- `disabled`

Comment (sethrosetter) [inline _queue/template.robot]: I like `unimplemented`. I will change it.

**Comment by sethrosetter** [comment]
Fixed

**Pull request merged**